### DISCUSSION TITLE
Strict Tracking (ST) mod Star Rating calculation.

### DISCUSSION THREAD

**Discussion by IkaWaAyuMu** [ppy/osu#27706] [Q&A] [answered]
Just want to know what is the criteria to increase Star Rating of a map when Strict Tracking (ST) mod if applied?

My assumption is the slider is hard to track.
https://osu.ppy.sh/beatmapsets/1585404#osu/3238282 (8.26 -> 10.01)
| NM | ST |
| --- | --- |
| ![image](https://github.com/ppy/osu/assets/7066150/8e1ae107-05a5-4962-8e6a-491a093b0b34) | ![image](https://github.com/ppy/osu/assets/7066150/c6a582c8-cd2c-47ce-ac01-374f4f535975) |

such as slider at `00:27:211 (1) - `

But so far, that's the only map I finds increase in SR

https://osu.ppy.sh/beatmapsets/1136149#osu/2404722 - `00:01:698 (1) - `
https://osu.ppy.sh/beatmapsets/460569#osu/986359 - A LOT

The others' slider is also hard(or impossible) to track, but no increase in SR at all.

**Comment by Walavouchey** [comment]
with strict tracking on, the slider at 00:27:212 has slider ticks while it normally doesn't. this is probably what affects the sr calculations

**Reply by IkaWaAyuMu** [reply]
![image](https://github.com/ppy/osu/assets/7066150/3d89cfb3-35f1-4b48-81a4-beedb88760b6)

I think the other one also have slider tick. https://osu.ppy.sh/beatmapsets/1136149#osu/2404722

**Reply by Walavouchey** [reply]
the other difference is that autoplay can ss the slider for the first map, but misses with strict tracking on

in the map you linked, autoplay can hit the slider regardless

**Comment by bdach** [comment, marked as answer]
Strict tracking has not been given proper difficulty consideration. Any star rating changes should be considered incidental and not reflective of the mod's true difficulty.
